scenario/utils.ts

```diff
@@ -238,4 +240,22 @@ export async function fetchQuery(
       return { err };
     }
   }
-}
\ No newline at end of file
+}
+
+export async function isValidAssetIndex(ctx: CometContext, assetNum: number): Promise<boolean> {
+  const comet = await ctx.getComet();
+  return assetNum < await comet.numAssets();
+}
+
+export async function isSourceable(ctx: CometContext, assetNum: number, amount: number): Promise<boolean> {
```

Comment:
Fancy! Technically like 'isSourceableFromFauceteer' then, but makes sense

Reply:
I actually originally named it something like that, but it felt weird because the function returns `true` if `fauceteer` doesn't exist on that chain.

Reply:
Yeah true, 'sourceable from fauceteer or fauceteer doesnt exist'...maybe `isTriviallySourceable`?

Reply:
Done